docs/guide/hot-swapping.md

```
@@ -1,5 +1,12 @@
 ## Hot swapping
 
+**📚 Documentation Index**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)